### PR TITLE
Fixing Kubernetes Pods Dashboard

### PR DESCRIPTION
**Pull request by kameshsampath**
This PR fixes the following:

* upgrading grafana to latest v4.0.2
* Kubernetes Dashboard not showing any data, and the pull down's not showing namespace/pods

TODO/WIP:

* Prometheus dashboard fixes

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-FROM grafana/grafana:2.6.0
-MAINTAINER Jimmi Dyson <jimmidyson@gmail.com>
+FROM grafana/grafana:latest
```
Comment (jimmidyson) [inline Dockerfile]: This should use a versioned tag.

Comment (kameshsampath) [reply Dockerfile]: @jimmidyson - we thought to rebase it with latest version always, or you want to nail it down to 4.0.2 ?

Comment (jimmidyson) [reply Dockerfile]: Not a good idea, ever, to track latest as you're then at the mercy of the from image provider ensuring backwards compatibility.

4.0.2 would be great.

**Comment by kameshsampath** [comment]
@jimmidyson check it now i have freezed the version to `4.1.1`

**Comment by jimmidyson** [comment]
LGTM thanks!

**Pull request merged**